translated_docs/zh-CN/run-locally.md

```
@@ -0,0 +1,159 @@
+此时，你应能在浏览器中通过 `http://captain.caproverinstance.local` 访问控制台。
+如果这里有问题，说明本地 DNS 服务器没有按预期工作。你必须先修复它。
+
+注意，你不应（也无法）为内部域名启用 HTTPS。
+
+### 选项 2 - 让实例可以从外部访问。
+
+要求：你的公网 IP 必须是静态 IP 地址。
+
+这和你在可公开访问的 VPS 上安装 CapRover 非常相似。你只需要在路由器上启用端口转发：
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)